string status
string last_status
int32 player_num
int32[] alive
string[] names
